### ISSUE TITLE
Bill of materials

### ISSUE THREAD

**Issue by UmangBR** [amosproj/amos2022ss03-turtlebot-fleet-management#5]
## User story
1. As a team
2. We need to initialize bill of materials with our dependencies
3. So that we have a centralized source of information needed for software development.

## Acceptance criteria
* Needs to be added on the planning documents

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
